examples/gallery/objects-in-splotlight.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+
+<head>
+	<style>
+		div {
+			font-family: Arial;
+		}
+	</style>
+</head>
+
+<body>
+	<main id="main"></main>
+	<script type="module">
+		import drawGraphics3d from '../../src/index.js';
+
+		drawGraphics3d(
+			document.getElementById('main'),
+			{
+				axes: {},
+				elements: [
+					{type: 'sphere',
+					 color: [1.0, 1.0, 1.0],
+					 opacity: 1.0,
+					 radius: 1,
+					 coords: [[[0.0, 0.0, 0.0]]]
+					},
+					{type: 'cuboid',
+					 color: [1.0, 1.0, 1.0],
+					 opacity: 1.0,
+					 coords: [[[1.0, 2.0, -1.0]],[[2.0, 3.0, 1.0]]]
+					},
+					{type: 'uniformPolyhedron',
+					 subType: "dodecahedron",
+					 color: [1.0, 1.0, 1.0],
+					 opacity: 1.0,
+					 coords: [[[3.0,-1.0, 0.0]]]}],
+				lighting: [
+					{type: 'directional',
```

Comment:
Maybe change the file name?

Reply:
By all means change the name to whatever makes sense. Thanks

And the reformatting is fine too. My Emacs isn't reformatting this HTML automatically. If tweaking the EditorConfig file will help that would be great.

Lastly, some of these mirror (or are better examples) of what is in the wiki. So the wiki examples and PNG's should be updated and links to the on-line versions I think would help.